src/gui/executionlogwidget.cpp

```diff
@@ -29,78 +29,75 @@
 #include "executionlogwidget.h"
 
 #include <QDateTime>
+#include <QMenu>
 #include <QPalette>
 
 #include "base/global.h"
-#include "loglistwidget.h"
+#include "base/logger.h"
```

Comment:
you already included this header in executionlogwidget.h so you don't need it here.

Reply:
removed seems like `base/global.h` was not needed too, removed as well.